src_modules/voice_commands/index.ts

```diff
@@ -1,9 +1,10 @@
 // extcord module 
 // requires fastest-levenshtein@^1.0.16 lru-cache@^10.0.1
 
-import { GatewayIntentBits, Guild, VoiceState } from "discord.js";
+import { GatewayIntentBits, Guild, VoiceState, VoiceConnection } from "discord.js";
```

Comment:
The VoiceConnection import needs to be from discordjs/voice, not from discord.js.